packages/nimble-components/src/rich-text/mention-list-box/index.ts

```diff
@@ -0,0 +1,288 @@
+import { Notifier, Observable, observable } from '@microsoft/fast-element';
+import {
+    DesignSystem,
+    ListboxElement as FoundationListbox,
+    ListboxOption
+} from '@microsoft/fast-foundation';
+import { keyEnter, keyEscape, keyTab } from '@microsoft/fast-web-utilities';
+import type { MentionDetail } from '../editor/types';
+import { styles } from './styles';
+import { template } from './template';
+import type { AnchoredRegion } from '../../anchored-region';
+import { diacriticInsensitiveStringNormalizer } from '../../utilities/models/string-normalizers';
+import type { ListOption } from '../../list-option';
+import type { MentionListBoxShowOptions } from './types';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-rich-text-mention-list-box': RichTextMentionListBox;
+    }
+}
+
+/**
+ * A rich text mention list box which acts as a popup for "@mention" support in editor
+ */
+export class RichTextMentionListBox extends FoundationListbox {
```

Comment:
We should match FAST's casing for "Listbox": all one word, not "list box" or "ListBox" or "list-box". This affects a few class and file names in the PR.

Reply:
Updated